pkg/manifests/types.go

```diff
@@ -753,6 +753,11 @@ type RemoteWriteSpec struct {
 	QueueConfig *monv1.QueueConfig `json:"queueConfig,omitempty"`
 	// Defines the timeout value for requests to the remote write endpoint.
 	RemoteTimeout string `json:"remoteTimeout,omitempty"`
+	// Enables sending of exemplars over remote write. Note that the experimental
+	// "exemplar-storage" feature must be enabled using the `spec.enableFeature`
+	// option for exemplars to be scraped in the first place. Only supported for
+	// UWM configurations.
```

Comment:
I'll omit the implementation details about the Prometheus command-line argument and rather focus on the capabilities and "limitations".

cc @bburt-rh for review.

```suggestion
	// Enables sending of exemplars over remote write. Note when enabled, Prometheus is
	// configured to store exemplars in memory with a maximum number of 100,000 exemplars. The setting is only effective for
	// user-defined monitoring and has no effect for cluster monitoring.
```